### ISSUE TITLE
QR-code doesnt show up on nixos 

### ISSUE THREAD

**Issue by TBS1996** [boxdot/gurk-rs#177]
Greetings, after i installed the program and run "gurk", the 'linking device' message comes up but nothing happens.

I have feh installed, do I need some other image viewer?

**Comment by boxdot** [comment]
Hi. The QR is rendered directly in the terminal. -- I just checked. For me, the QR code opens almost immediately.

**Comment by TBS1996** [comment]
hmm, I tried with kitty but it still didnt work, ill see if it works on manjaro later perhaps

**Comment by TBS1996** [comment]
Tried on manjaro and it worked, so i dont know whats causing this. Ive uploaded my flake to here: https://github.com/TBS1996/nix-flake

**Comment by cleeyv** [comment]
I wasn't able to reproduce this bug on NixOS. It worked fine on both gnome-console and kitty. Based on your config I see that you're using Sway WM so I tried that as well and it worked with the default terminal emulator (foot). I also tried it in a virtual console (ctrl-alt-f1) and it worked, though the QR code was rendered with a mix of grey blocks, and black and white diamond characters, and it couldn't be read by the Signal QR code scanner. 

I see in your flake.nix that you've set nixpkgs to nixos-unstable, but maybe this hasn't successfully updated all your packages for some reason? What result are you getting from `gurk --version`? I'm quite sure I have the latest version 0.2.4 installed, but when I run `gurk --version` I see 0.2.3, and when I run the 22.05 version I see 0.2.2 even though I'm pretty sure it is actually 0.2.3. This discrepancy seems to be some kind of off-by-one error with the `--version` number, but that would be a different issue.
